test/unit/helpers/help_helper_test.rb

```diff
@@ -37,4 +39,31 @@ class HelpHelperTest < ActionView::TestCase
     assert_equal 1,tag.children.count
     assert_equal 'Help', tag.children.first.content
   end
+
+  test 'help icon' do
+    link = index_and_new_help_icon 'collection'
+    tag = Nokogiri::HTML::DocumentFragment.parse(link).children.first
+    
+    assert_equal 'https://docs.seek4science.org/help/user-guide/collections.html', tag['href']
+    assert_equal '_blank', tag['target']
+    assert_equal t("info_text.collection"), tag['data-tooltip']
+    assert_equal 'What is a Collection?', tag.children[1].content
```

Comment:
Would be good to also have a test that proves the `indefinite_article` method works for e.g. "an Assay".

Reply:
Really good idea and added :)